.env.production.example

```
@@ -0,0 +1,42 @@
+VITE_LOGIN_URL=
+VITE_GITHUB_APP_INSTALL_URL=https://github.com/apps/raw-gent/installations/new
+
+COOKIE_DOMAIN=
+COOKIE_PATH=/
+COOKIE_SECURE=true
+COOKIE_SAMESITE=lax
+COOKIE_ACCESS_NAME=access_token
+COOKIE_INSTALLATION_NAME=installation_id
+
+REDIS_URL=redis://redis:6379/0
+REDIS_SSL_VERIFY=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REDIS_SSL_VERIFY key should go before the REDIS_URL key

(UnorderedKey)